sotodlib/preprocess/core.py

```diff
@@ -30,6 +31,7 @@ def __init__(self, step_cfgs):
         self.calc_cfgs = step_cfgs.get("calc")
```

Comment:
For the doctoring of `_Preprocess` also add a mention to `plot` where it describes the list of keys that are matched to functions in the module.